dbms/src/Storages/DeltaMerge/ColumnFile/ColumnFileSetReader.h

```diff
@@ -33,6 +33,8 @@ class ColumnFileSetReader
     ColumnDefinesPtr col_defs;
     RowKeyRange segment_range;
 
+    LACBytesCollector lac_bytes_collector;
```

Comment:
how about moving to L44?
I'm a bit worried that `ColumnFileReader`'s destructor might take some time.

Reply:
ok